pokedex_async_redux/lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -82,4 +155,23 @@ class PokemonOverviewPage extends StatelessWidget {
       MaterialPageRoute(builder: (_) => PokemonDetailsConnector(pokemon: pokemon)),
     );
   }
+
+  void _clearSearchedPokemons() {
+    widget.clearSearchedPokemons;
+    setState(() {
```

Comment:
use fat arrow notation for this setState since you're just gonna update a single property 🙂 

`setState( () => isSearching = false);`